tests/test_pandas_extension.py

```diff
@@ -169,6 +229,36 @@ def test_reindex(self):
         # due to a dtype assumption that does not hold for Arrow
         pass
 
+    def test_get(self, data, dtype):
+        if pa.types.is_list(dtype.arrow_dtype):
+            pytest.skip("assert_extension_array_equal casts to numpy object arrays")
```

Comment:
Maybe use xfail in case the tests are ever rewritten we'd notice

Reply:
As there is a general `xfail` for list types, I have removed all the test specializations.